RR.UnitOfWork/SEED_DUMMY.sql

```diff
@@ -649,4 +649,7 @@ WHERE id = 11;
 UPDATE "Employee" SET "engagementDate" = TIMESTAMPTZ '2024-03-11 06:48:18.410491Z'
 WHERE id = 12;
 UPDATE "Employee" SET "engagementDate" = TIMESTAMPTZ '2024-03-11 06:48:18.410492Z'
-WHERE id = 13;
\ No newline at end of file
+WHERE id = 13;
+
+INSERT INTO "EmailTemplate"(name, subject, body)
+VALUES ('WelcomeLetter', 'Welcome to Retro Rabbit, {{Name}}', '<html><body><p>Dear {{Name}},</p><p>We are thrilled to have you on board! Welcome to the Retro family.</p><p>From today, you embark on a new journey with us, filled with exciting opportunities, challenges, and growth. At Retro, we pride ourselves on fostering a culture of collaboration, innovation, and mutual respect. We believe that every individual brings a unique perspective and talent to the team, and we can''t wait to see the wonderful contributions you''ll make.</p><p>Remember, it''s okay to feel overwhelmed or have questions. We''ve all been there. Don''t hesitate to ask or seek clarification on anything. Our doors (and inboxes) are always open.</p><p>Your assigned recruiter and a representative from our Journey team will reach out to you within the next few days to assist with and facilitate the upcoming onboarding process.</p><p>Once again, welcome to Retro Rabbit. Here''s to new beginnings and the start of a memorable journey together!</p><p>Click <a href=''https://hris-client.azurewebsites.net''>here</a> to visit our employee portal.</p></body></html>');
```

Comment:
Future:
Readability and maintainability by separating the email body 
construction into smaller functions: GreetingMessage, WelcomeMessage, and 
OnboardingInfo. These functions can be customized independently and reused for 
building other email templates if needed.

Additionally, the use of placeholders (?) and parameterized queries enhances security 
by preventing SQL injection vulnerabilities.

INSERT INTO "EmailTemplate" (name, subject, body)
VALUES (
  'WelcomeLetter',
  'Welcome to Retro Rabbit, ?',
  CONCAT(
    '<html><body>',
    '<p>Dear ', ?, ',</p>',
    -- Other message sections as separate variables
    GreetingMessage(?),
    WelcomeMessage(),
    OnboardingInfo(),
    '</body></html>'
  )
);

Reply:
We won't be breaking the email templates up into smaller components. Each email will be stored individually. A future approach on this would be to have a email building for the end user to use.

There are no SQL injection concerns. The details will not be used on a SQL level and the placeholders `{{Name}}` is for the compiling of object data into the string using Handlebars package.